HISTORY.md

```diff
@@ -1,6 +1,6 @@
 ## v3.0.0
 
-- Meteor 1.6
+- Meteor 1.6.1
```

Comment:
We need to bump the version of the image.  We can't just add this to a previous release.

Reply:
And this should be 1.6.0.1